spec/rubocop/cop/naming/predicate_name_spec.rb

```diff
@@ -145,4 +145,43 @@ def is_a?; end
       RUBY
     end
   end
+
+  context 'using Sorbet sigs' do
```

Comment:
I think you should also add tests verifying the behavior in the absence of a signature when this config is enabled.